docs/management/dashboard_only_mode/advanced_configuration.asciidoc

```diff
@@ -1,19 +1,24 @@
 [role="xpack"]
 [[advanced-dashboard-mode-configuration]]
-=== Advanced configuration for Dashboard Only mode
+=== Advanced Configuration for Dashboard Only Mode
 
-Kibana comes preconfigured with a reserved role called `kibana_dashboard_only_user` which will be sufficient
-for most use cases.  However, if your setup requires a custom Kibana index, you can manually create
-Dashboard Mode roles.
+Kibana comes preconfigured with a reserved role called `kibana_dashboard_only_user` 
+that grants read only access to the `.kibana` index. This role is sufficient
```

Comment:
Maybe it's obvious, but it might be worth restating that this role is only available to you if security is enabled.